src/app/components/ContactModal.js

```diff
@@ -120,6 +122,7 @@ const ContactModal = ({ contactID, properties: initialProperties, ...rest }) =>
         const notEditableProperties = initialProperties.filter(({ field }) => !EDITABLE_FIELDS.includes(field));
         const Contacts = await prepareContacts([properties.concat(notEditableProperties)], userKeysList[0]);
         await api(addContacts({ Contacts, Overwrite: +!!contactID, Labels: 0 }));
+        call();
```

Comment:
Make sure everything is sync before to display the success notification.
```suggestion
        await call();
```